src/utils/openwhisk.js

```diff
@@ -90,6 +90,7 @@ function extractActionContext(params) {
     __ow_headers,
     __ow_method,
     __ow_logger,
+    body, // skip body here
```

Comment:
I would keep it there. in case the `body` doesn't provide the `content.body` but other data that might be useful.